README.md

```diff
@@ -67,6 +67,9 @@ First, you're going to need to copy the `.env.example` to `.env`. The current op
 | `EMAIL_USERNAME` | Gmail address; e.g. `jensen.robbed.us@gmail.com` |
 | `EMAIL_PASSWORD` | Gmail password; see below if you have MFA |
 | `STORES` | List of [stores](#Supported-stores) you want to be scraped; optional, default: `nvidia` |
+| `PHONE_NUMBER` | 10 digit phone number. ONLY FOR USA. SMS CHARGES APPLY, for example: `123456789` |
```

Comment:
> 10 digit phone number

Uses 9 digit phone number as example 🤣 

Seriously note, look great!!!

Reply:
🤣  good catch! and thank you :)